core/thread/src/TImplicitMT.cxx

```diff
@@ -20,15 +20,13 @@
 
 #include "TError.h"
 #include "TThread.h"
-
+#include "ROOT/TPoolManager.hxx"
 #include <atomic>
-#include "tbb/task_scheduler_init.h"
-
 
-static tbb::task_scheduler_init &GetScheduler()
+static std::shared_ptr<ROOT::TPoolManager> &GetPoolManagerMT()
 {
-   static tbb::task_scheduler_init scheduler(tbb::task_scheduler_init::deferred);
-   return scheduler;
+   static std::shared_ptr<ROOT::TPoolManager> schedMT;
```

Comment:
humm .. I see, I missed the GetPoolManager() that does what I expect.   Since this is an internal utility function please consider renaming R__GetPoolManagerMT.